app/services/github_client/user.rb

```diff
@@ -4,11 +4,23 @@ def initialize(username)
       @username = username
     end
 
-    def repositories
-      response = connection.get("/users/#{@username}/repos") do |request|
-        request.params['type'] = 'member'
+    def pull_request_events
+      events = (1..10).map do |page|
+        response = connection.get("/users/#{@username}/events/public") do |request|
+          request.params['page'] = page
+        end
+        results = JSON.parse(response.body, symbolize_names: true)
+
+        if results.empty? || older_than_a_day(results.last[:created_at]) || results.size < 30
```

Comment:
Why that `30`? Is it the default number of results the endpoint returns? Could we extract it to a constant?

Reply:
I will move it to a constant. Its the default (and unchangable) per page value